package.json

```diff
@@ -50,7 +50,9 @@
     "node-mocks-http": "1.2.3",
     "sinon": "1.17.6",
     "typescript": "2.0.10",
-    "typings": "2.0.0"
+    "@types/mocha": "^2.2.40",
+    "@types/node": "^7.0.12",
```

Comment:
Do you want to use the latest typings, or the typings for the earliest node that you support? By using the latest, you might not notice if you use a newer language feature (e.g. `array.find`).